### ISSUE TITLE
Fullpage diff

### ISSUE THREAD

**Issue by alanpilloud** [segmentio/niffy#21]
**What needs to get done?**

I'm trying to solve issue #18 (taking fullpage screenshots). I've managed to capture fullpage screenshots. However, the diff doesn't run up to 2144px of height.

Anyone has an idea why the diff does not process the whole page ?

**Why does it need to get done?**

- That would solve #18 
- I think that is a very important feature.

**How do I do it?**

<details>
  <summary>Setup</summary>

  - Clone the repo by running `git clone https://github.com/alanpilloud/niffy`
  - Run `npm install`
  - Run tests with `make tests` (tests will "fail" for now, as expected)
  - Go check the diff for the page /services
  
</details>

**Comment by alanpilloud** [comment]
I've tried to implement modifications in the pull request #20 and now it is working :smile: 
I'll open a new pull request indicating that it is dependent on #20
